media/Config/exportConfig.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var importConfigPath = '';
+
+// Get file name from the path.
+const getFileName = function(oneToolList) {
+  let filePath = importConfigPath;
+  // The default name of '.cfg' file will be changed depending on
+  // the model file or imported '.cfg' file.
+  if (filePath === '') {
```

Comment:
I can't find `importConfigPath` in your draft PR. I'd like to look where `importConfigPath` is updated.

Reply:
You can find `importConfigPath` at line 17 in `media/Config/exportConfig.js` 
And it will be used at line 41 in `media/Config/receiveFromPanel.js`

Reply:
Then `importConfigPath` is always `''` ? meaning `if (filePath === '')` will be always `True` ?

Reply:
> Then `importConfigPath` is always `''` ? meaning `if (filePath === '')` will be always `True` ?

No it's not. If the user imports the `.cfg` file, then `importConfigPath` will have a path of the file.
This will work at line 41 in `media/Config/receiveFromPanel.js`
But if the user doesn't import the `.cfg` file, then `importConfigPath` will remain as `''`

I think you're confused because I'm using `importConfigPath` as a global variable.
As I defined `importConfigPath` here once, it can be accessed from any `.js` files.
That's why `importConfigPath` won't be always `''`.

Reply:
> That's why importConfigPath won't be always ''.

OK got it, thanks!